[CF1] update tunnel instructions (#19251)
cloudflare/cloudflare-docs: src/content/docs/cloudflare-one/connections/connect-networks/get-started/create-remote-tunnel.mdx

pcx_content_type: how-to
sidebar:
  order: 1
---

import { Render } from "~/components";

Follow this step-by-step guide to get your first tunnel up and running using Zero Trust.

## 1. Create a tunnel

<Render file="tunnel/create-tunnel" product="cloudflare-one" />

> The next steps depend on whether you want to [connect an application](#2-connect-an-application) or [connect a network](#3-connect-a-network).

Reviewer: ```suggestion
The next steps depend on whether you want to [connect an application](#2a-connect-an-application) or [connect a network](#2b-connect-a-network).
```

## 2a. Connect an application

Before you connect an application through your tunnel, you must:

- [Add a website to Cloudflare](/fundamentals/setup/manage-domains/add-site/).
- [Change your domain nameservers to Cloudflare](/dns/zone-setups/full-setup/setup/).

> Follow these steps to connect an application through your tunnel. If you are looking to connect a network, skip to the [Connect a network section](#3-connect-a-network).

Reviewer: ```suggestion
Follow these steps to connect an application through your tunnel. If you are looking to connect a network, skip to the [Connect a network section](#2b-connect-a-network).
```

<Render file="tunnel/add-public-hostname" product="cloudflare-one" />

> The application is now publicly available on the Internet. To allow or block specific users, [create an Access application](/cloudflare-one/applications/configure-apps/self-hosted-public-app).

Reviewer: ```suggestion
The application is now publicly available on the Internet. To allow or block specific users, [create an Access application](/cloudflare-one/applications/configure-apps/self-hosted-public-app/).
```

> The application is now publicly available on the Internet. To allow or block specific users, [create an Access application](/cloudflare-one/applications/configure-apps/self-hosted-public-app).

Author: ```suggestion
The application is now publicly available on the Internet. To allow or block specific users, [create an Access application](/cloudflare-one/applications/configure-apps/self-hosted-public-app/).
```
